pkg/pillar/types/patchenvelopestypes.go

```diff
@@ -86,3 +126,24 @@ type BinaryBlobVolumeRef struct {
 	ArtifactMetadata string `json:"artifactMetaData"`
 	ImageID          string `json:"imageId"`
 }
+
+// PatchEnvelopeUsage stores information on how patchEnvelopes are
+// used by App Instances to send this information back to controller
+// reflects ZInfoPatchEnvelopeUsage proto message
+type PatchEnvelopeUsage struct {
+	AppUUID string
+	PatchID string
+	Version string
+	// count the number of times app instance called patch APIs
+	PatchAPICallCount uint64
+	// count the number of times app instance actually downloaded
+	// whole patch envelope or part of it
+	TimesDownload uint64
```

Comment:
You tweaked this name in the API, so might make sense to do the same tweak in this struct.

Reply:
Changed the name